docker-compose.yml

```diff
@@ -1,47 +1,56 @@
-version: "3.0"
+version: "3.5"
+
 services:
   database:
     image: amsterdam/postgres11
     ports:
-      - "5409:5432"
+      - 5432
     environment:
-      POSTGRES_DB: blackspots
-      POSTGRES_USER: blackspots
-      POSTGRES_PASSWORD: insecure
-    volumes:
-      - "~/.ssh/datapunt.key:/root/.ssh/datapunt.key"
```

Comment:
Ik weet niet zeker of en hoe die werd gebruikt.. Moeten we dubbelchecken voordat we mergen.

Reply:
ik laat hem nog ff staan